src/main/java/com/banking/application/repository/RoleRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.banking.application.repository;
+
+import com.banking.application.entity.Role;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface RoleRepository extends JpaRepository<Role, Long> {
+    @Query(value = "from roles where roleName = ?1")
+    Optional<Role> getByName(Role.RoleName name);
```

Comment:
```suggestion
    Optional<Role> getByRoleName(Role.RoleName name);
```
Spring Data JPA is capable of automatically build the query in many cases based on the method name and parameters)